build/yaml/skillFunctionalTestsBuild.yml

```diff
@@ -0,0 +1,40 @@
+steps:
```

Comment:
rename this file as dotnetBotsBuild-CI.yml

Reply:
renamed to dotnetBotsBuild-CI.yml